indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -47,6 +68,9 @@
 
   private static final Set<Path> existing = Sets.newHashSet();
 
+  private static final int NUM_RETRIES = 6;
```

Comment:
we actually have 10, but with exponential backoff in S3DataSegmentPusher, maybe we can use the same here?